speech-commands/training/browser-fft/training_custom_audio_model_in_python.ipynb

```diff
@@ -0,0 +1,815 @@
+{
```

Comment:
<p>I think the questions is if I have a set of .wav files, what part of this script needs to be changed?</p><p> </p><p>Better if this can take a custom directory and require no code changes from user.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tfjs-models/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Good question. I added a paragraph in this header section to answer that question.</p>